### PR TITLE
refactor: fix tailwind anti-pattern in feed footer

### LINKED ISSUE #2085: Feature: refactor tailwind anti-pattern from feed(index.tsx), footer, notification and search-dialog components
### Type of feature

🧑‍💻 Refactor

### Current behavior

 I've identified some anti-patterns in the styling logic for the feed(index.tsx), footer, notification, and search-dialog components, specifically related to the usage of Tailwind CSS. I would like to undertake the task of refactoring these components to ensure that the code adheres to best practices and follows the established guidelines for efficient Tailwind CSS usage.
 
 This is related to this issue: https://github.com/open-sauced/app/issues/1522 and #2053

### Suggested solution

I will replace the anti-pattern with a matching value in the tailwind classes.

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by RitaDee**
## Description

This PR addresses the Tailwind CSS anti-patterns in the feed(index.tsx), footer, notification, and search-dialog components by simplifying the styling logic. 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [x] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Fixes #2085

## Mobile & Desktop Screenshots/Recordings

Before:

https://github.com/open-sauced/app/assets/106407013/46f3b6ba-e982-4d2f-b3f1-c1032152d860

After:

https://github.com/open-sauced/app/assets/106407013/309dcfaa-dc57-4091-b185-c3bc3180ace4

https://github.com/open-sauced/app/assets/106407013/ef916fc1-8805-47fe-badc-898d0ea47396

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/organisms/SearchDialog/search-dialog.tsx [hunk 1]
```diff
@@ -70,13 +70,10 @@ const SearchDialog = () => {
   }, [debouncedSearchTerm]);
 
   return (
-    <div className="fixed left-0 top-0 z-[55] p-5 w-full h-full flex justify-center bg-white/30">
+    <div className="fixed left-0 top-0 z-auto p-5 w-full h-full flex justify-center bg-white/30">
```
Comment (OgDev-01) [inline components/organisms/SearchDialog/search-dialog.tsx:73]: Wondering if this won't cause a breaks somewhere in the app 🤔

**Review by OgDev-01 — APPROVED** [review]
This looks good... just need a before/after screenshot ⚡️

File: components/organisms/SearchDialog/search-dialog.tsx (hunk 1 above)
Comment (RitaDee) [reply components/organisms/SearchDialog/search-dialog.tsx:73]: I will do a video on what the UI looks like now.

**Comment by nickytonline** [comment]
@RitaDee, thanks for working on this. Can you share some before and after screenshots, or do the units with the Tailwind classes that replaced the units match up 100%?

**Review by nickytonline — APPROVED** [review]
Thanks for the cleanup @RitaDee! :shipit:

**Pull request merged**